website/profile/views.py

```diff
@@ -353,9 +354,10 @@ def user_account_password(auth, **kwargs):
         user.change_password(old_password, new_password, confirm_password)
         user.save()
     except ChangePasswordError as error:
-        push_status_message('<br />'.join(error.messages) + '.', kind='warning')
+        for m in error.messages:
+            push_status_message(m, kind='warning', trust=False)
```

Comment:
This looks like it will result in a series of error messages rather than one message. This is inconvenient, and may actually be a regression. Can we combine these without using HTML rather than having multiple?